pyproject.toml

```
@@ -384,9 +439,8 @@ ignore = [
 ]
 extend-exclude = ["CommonServerPython", "test_data"]
 
-# Allow autofix for all enabled rules (when `--fix`) is provided.
 # fixable = []
-# unfixable = []
+unfixable = ["RUF001"]
```

the name of the check is missing 
```suggestion
unfixable = [
    "RUF001" # ambiguous-unicode-character-string
]
```